### ISSUE TITLE
Request accessibility/508 review for SCO Migration

### ISSUE THREAD

**Issue by desireeturner** [department-of-veterans-affairs/va.gov-team#3045]
## Team Name
BAH

## Product Description
Resources for schools is a one-stop shop for School Certifying Officials (SCOs) and school administrators assisting students who are using their VA benefits to pursue education and training programs. Find trainings, resources, guides, and information on GI Bill programs to support military-connected students.

## Staging URL and Credentials

https://staging.va.gov/school-administrators/

## Test Plan
Static content page, so no test plan. Let us know if you need one. 

## Known Issues

- [508- SCO - Axe check failure: Heading levels should only increase by one #19968](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19968)
